detox/test/e2e/__snapshots__/06.device.test.js.snap

```diff
@@ -0,0 +1,61 @@
+}
+`;
+
+exports[`Device getViewHierarchyXml() - should return a valid UI hierarchy: android RN:73 1`] = `
+{
+  "result": "<?xml version='1.0' encoding='UTF_8' standalone='yes' ?><ViewHierarchy><DecorView class="com.android.internal.policy.DecorView" width="1080" height="2220" visibility="visible" x="0" y="0" alpha="1.0" focused="false" value="" label="null" testID=""><LinearLayout class="android.widget.LinearLayout" width="1080" height="2088" visibility="visible" x="0" y="0" alpha="1.0" focused="false" value="" label="null" testID=""><ViewStub id="android:id/action_mode_bar_stub" class="android.view.ViewStub" width="0" height="0" visibility="gone" x="0" y="0" alpha="1.0" focused="false" value="" label="null" testID="" /><FrameLayout class="android.widget.FrameLayout" width="1080" height="2022" visibility="visible" x="0" y="66" alpha="1.0" focused="false" value="" label="null" testID=""><FitWindowsLinearLayout id="com.wix.detox.test:id/action_bar_root" class="androidx.appcompat.widget.FitWindowsLinearLayout" width="1080" height="2022" visibility="visible" x="0" y="66" alpha="1.0" focused="false" value="" label="null" testID=""><ViewStubCompat id="com.wix.detox.test:id/action_mode_bar_stub" class="androidx.appcompat.widget.ViewStubCompat" width="0" height="0" visibility="gone" x="0" y="66" alpha="1.0" focused="false" value="" label="null" testID="" /><ContentFrameLayout id="android:id/content" class="androidx.appcompat.widget.ContentFrameLayout" width="1080" height="2022" visibility="visible" x="0" y="66" alpha="1.0" focused="false" value="" label="null" testID=""><ReactRootView id="11" class="com.facebook.react.ReactRootView" width="1080" height="2022" visibility="visible" x="0" y="66" alpha="1.0" focused="false" value="" label="Hello!!!" testID=""><ReactTextView id="289" class="com.facebook.react.views.text.ReactTextView" width="213" height="92" visibility="visible" x="434" y="1059" alpha="1.0" focused="false" value="" label="Hello!!!" testID="" text="Hello!!!" /></ReactRootView></ContentFrameLayout></FitWindowsLinearLayout></FrameLayout></LinearLayout><View id="android:id/navigationBarBackground" class="android.view.View" width="1080" height="132" visibility="visible" x="0" y="2088" alpha="1.0" focused="false" value="" label="null" testID="" /><View id="android:id/statusBarBackground" class="android.view.View" width="1080" height="66" visibility="visible" x="0" y="0" alpha="1.0" focused="false" value="" label="null" testID="" /></DecorView></ViewHierarchy>",
```

Comment:
Can we somehow add empty lines and indentations like on iOS? (pretty print)

Reply:
hmm couldn't find this config in the [XmlSerializer](https://developer.android.com/reference/org/xmlpull/v1/XmlSerializer) API. Only if you find easy way to do it - not really important!